### PR TITLE
5 api users

### PR DESCRIPTION
**Pull request by olivershh**

### CODE REVIEW

**Comment by olivershh** [comment]
Already had a bunch of file changes from yesterday that shouldn't really have been in here. I promise to keep it cleaner next time!

File: __tests__/app.test.js [hunk 1]
```diff
@@ -84,3 +84,34 @@ describe("/api/articles/:article_id", () => {
...
+        });
+    });
+    test("200: returns array of user objects with correct properties and data types", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then((response) => {
+          const { body } = response;
+
+          expect(body.users.length !== 0).toBe(true);
```
Comment (Poonam-raj) [inline __tests__/app.test.js:106]: You should be way more specific about your length check here - you know how many users you should get back

**Review by Poonam-raj — APPROVED** [review]
Looks all good - only amendment needed is the length check in your test file but once you've fixed that feel free to merge

**Comment by olivershh** [comment]
When I'm making generic changes (renaming files, refactoring, moving error handling to a separate controller etc.) that don't fit into a specific ticket, do I still need to get them approved for a merge? Or am I ok to just make those kind of changes on main without checking in on them?

File: __tests__/app.test.js (hunk 1 above)
Comment (olivershh) [reply __tests__/app.test.js:106]: Ok good point, thanks.

**Pull request merged**